pkg/configure/oneagent/pmc/create.go

```diff
@@ -0,0 +1,62 @@
+package pmc
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/pmc/ruxit"
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+func Create(log logr.Logger, fs afero.Fs, source, destination string, conf ruxit.ProcConf) error {
```

Comment:
```suggestion
func Create(log logr.Logger, fs afero.Fs, srcFile, dstFile string, procConf ruxit.ProcConf) error {
```
more redundancy, but IMO easy to read and less error prone in usage